src/core/scene/a-scene.js

```diff
@@ -65,11 +65,30 @@ module.exports = registerElement('a-scene', {
           this.setupRenderer();
           this.resize();
         });
+        this.addFullScreenStyles();
         initPostMessageAPI(this);
       },
       writable: true
     },
 
+    addFullScreenStyles: {
+      value: function () {
+        var htmlEl = document.querySelector('html');
+        htmlEl.classList.add('a-html');
+        document.body.classList.add('a-body');
```

Comment:
a-html is only for non embedded use cases. I expect to see on the page very little else besides the a-frame scene.